gui/src/app/facilities/facilities-detail/facilities-detail.component.html

```diff
@@ -44,20 +44,20 @@ <h4 class="mt-4">{{'FACILITIES.FACILITY_ATTRIBUTES' | translate}}</h4>
         <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
         <tr mat-row *matRowDef="let row; columns: displayedColumns;" class="hover-dark"></tr>
       </table>
-
-      <button class="ml-2" mat-raised-button color="accent" (click)="deleteFacility()">
-        {{'FACILITIES.DELETE_FACILITY' | translate}}
-      </button>
-      <button class="ml-2" mat-raised-button color="accent" (click)="editFacility()">
-        {{'FACILITIES.EDIT_FACILITY' | translate}}
-      </button>
-      <button class="ml-2" mat-raised-button color="accent" (click)="addFacilityAdmin()">
-        {{'FACILITIES.ADD_ADMIN' | translate}}
-      </button>
-      <button class="ml-2" mat-raised-button color="accent" (click)="editAdmins()">
-        {{'FACILITIES.EDIT_ADMIN' | translate}}
-      </button>
-
+      <div class="mt-4">
+        <button class="ml-2" mat-raised-button color="accent" (click)="deleteFacility()">
```

Comment:
You should open a modal to let the user confirm if he truly wants to remove the facility. Also, the color of this button should be 'warning'.

Reply:
Also, the modal should not contain only 'are you sure?' but it should describe exactly, what the user is about to do. E.g. 'Are you sure you want to remove facility 'XYZ'?